tests/verify_error_messages_while_sigining_in.js

```diff
@@ -0,0 +1,72 @@
+const Signin_Page = require('../PageObjects/SignIn.Page')
+const Common= require('../common_files/Common_Fields')
+
+beforeEach (()=>
+{
+    Common.OpenBrowser()
+})
+
+describe('Verifying error messages While signing in', ()=>
+{    
+    it('With Blank Email Address and Valid Password',()=>
+    {
+        Signin_Page.Password_Textbox.setValue(Common.Valid_Password)
+        Signin_Page.Login_Button.click()
+
+        Common.Wait(1000)
+        expect(Common.ErrorMessage.isVisible()).to.true
+        expect(Common.ErrorMessage.getText()).to.equal(Common.Invalid_ErrorMessage)            
```

Comment:
When you are verifying text for the Error Message it means that it is visible on page. There is no requirement to verify that again on line no.17.